src_py/apiServer/transmitter.py

```diff
@@ -120,7 +118,7 @@ def contPhase(self, phase):     # phase can be train/training no matter capitals
         print("starting additional training")
 
         globe.pendingAcks += 1
-        if(phase.lower() == "train" or phase.lower() == "training"):
+        if(phase.lower().startswith("t")):      # t for train / training
```

Comment:
This is not a good idea to count on a single letter. 
why not startswith "train" which captures both train and training?

Reply:
I just wanted to shorten train / training as much as possible. Doesn't really matter since this is not live user input but API code, changing to startswith("train")